### ISSUE TITLE
Korath exile stations will fine you

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#2059]
http://steamcommunity.com/app/404410/discussions/0/282992646973682364/#c143388380482598903
> One interesting thing to note, is that one of the Raider worlds will fine you for having illegal outfits!

I'm not sure which world(s) in particular this applies to, though.

I haven't actually checked it myself :|

**Comment by Amazinite** [comment]
```
planet "Far'en Lai"
	landscape land/dune3
	description `When the Korath were banished to the Core, they were left with only one living world, a planet they call "Far'en Lai," meaning, "the last candle-flame." Afraid that this planet's biosphere will turn to dust in their hands like all the other worlds they once controlled, the Korath have left Far'en Lai entirely unsettled and unexploited. It is rumored that the terms of their exile dictate that if Far'en Lai withers and dies, the remaining Korath exiles will be exterminated, but that if they can learn to make the planet flourish, their place in the galaxy will be restored.`
	government Uninhabited
```

I guess the Uninhabited government hates nerve gas.

**Comment by Bladewood** [comment]
*shrug*

**Comment by Amazinite** [comment]
Pretty sure it should have `attributes uninhabited` along with the Uninhabited government, or security 0 as to not fine you.

**Comment by mootootwo** [comment]
No, it is the military station:
`planet "Kessel Sepret"
	attributes korath station
	landscape land/station1
	description `In this factory station, raw metal harvested in the other Korath systems is shaped into ships and weapons, the war materiel for their raids on Human space. Sections of the ships are assembled and tested in pressurized hangars, but the Korath Raider is a large enough ship that the final assembly must be done in space, outside the station.`
	spaceport `This station is run with military discipline reminiscent of a human Navy base. All the Korath, even the construction workers, are in full uniform with some sort of rank insignia: epaulettes that vary both in shape and color. The largest ones appear to mark the officers in charge.`
`
it is missing the `security 0` flag that other Korath worlds have.  Possibly intentional, as it is a military station of sorts.

**Comment by KiLEdEnNis** [comment]
if a planet has no security tag does it always check?

**Comment by mootootwo** [comment]
I have gotten it inconsistently on the Korath Rebel worlds and stations.  It must have a % chance to detect?  I don't have and scramblers or interface plate, so that shouldn't be stopping it.

The fine occurred on two different worlds, but re-landing on them about 5 times each never caused it to repeat.  The first world it happened on, I had never visited before.  The second world it happened on, I was returning to re-test landing on all the worlds (so I had landed there without triggering a fine previously)

**Comment by endless-sky** [comment]
Fixed in 5865248.

**Comment by endless-sky** [comment]
Closing because 0.9.6 is released.
